src/components/login/loginForm.tsx

```diff
@@ -1,73 +1,125 @@
-import './login.css';
 import { signInWithEmailAndPassword } from 'firebase/auth';
-import { useState } from 'react';
 import { auth } from '../../firebase-config';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import Link from '@mui/material/Link';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
 
-function Login() {
-    const [email, setEmail] = useState('');
-    const [password, setPassword] = useState('');
+function Copyright(props: any) {
+  return (
+    <Typography variant="body2" color="text.secondary" align="center" {...props}>
+      {'Copyright © '}
+      <Link color="inherit" href="https://mui.com/">
+      https://agilecoderquest.chrisuribe.com/
+      </Link>{' '}
+      {new Date().getFullYear()}
+      {'.'}
+    </Typography>
+  );
+}
 
-  const signIn = (e: { preventDefault: () => void; }) => {
+function Login() {
+  const signIn = (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
+    const form = e.currentTarget;
+    const data = new FormData(e.currentTarget);
+    const email = data.get('email');
+    const password = data.get('password');
+
+if (typeof email === 'string' && typeof password === 'string') {
     signInWithEmailAndPassword(auth, email, password)
     .then((userCredential) => {
-      console.log(userCredential)
+      form.reset();
+      const user = userCredential.user;
+      console.log('User logged in:', user);
     }).catch((error) => {
-      console.log(error)
-    })
+      form.reset();
+      const errorCode = error.code;
+      const errorMessage = error.message;
+      console.error('Error loggin in:', errorCode, errorMessage);
```

Comment:
Instead of a console.error, lets let the user know there was an issues logging in by using a better notification (Called a snackbar in React's MUI, but more commonly a 'toast notification'). 
The docs can be found here with examples:
https://mui.com/material-ui/react-snackbar/

Reply:
toast added